vmm/src/cpu.rs

```diff
@@ -1247,6 +1247,16 @@ impl CpuManager {
             .collect()
     }
 
+    #[cfg(target_arch = "aarch64")]
+    pub fn get_presented_mpidrs(&self) -> Vec<u64> {
```

Comment:
```suggestion
    pub fn online_mpidrs(&self) -> Vec<u64> {
```
I think the name of this function shouldn't include `get` as it's implicit. And I'd prefer `online` instead of `presented` since you rely on the list of onlined vCPUs to get the corresponding mpidrs.